.github/workflows/e2e-test-workflow-call.yml

```diff
@@ -253,3 +258,49 @@ jobs:
           name: '${{ matrix.entrypoint }}-${{ matrix.test }}'
           path: e2e/diagnostics
           retention-days: 5
+
+  e2e-test-suites:
+    # temporary flag. eventually this field will not exist and this will be the default.
+    if: ${{ inputs.temp-run-full-suite }}
+    runs-on: ubuntu-latest
+    needs:
+      - build-test-matrix
+      - docker-build
+      - docker-build-wasm
+    env:
+      CHAIN_IMAGE: '${{ inputs.chain-image }}'
+      CHAIN_A_TAG: '${{ inputs.chain-a-tag }}'
+      CHAIN_B_TAG: '${{ inputs.chain-b-tag }}'
+      RELAYER_IMAGE: '${{ inputs.relayer-image }}'
+      RELAYER_TAG: '${{ inputs.relayer-tag }}'
+      RELAYER_ID: '${{ inputs.relayer-type }}'
+      CHAIN_BINARY: '${{ inputs.chain-binary }}'
+      CHAIN_UPGRADE_TAG: '${{ inputs.chain-upgrade-tag }}'
+      CHAIN_UPGRADE_PLAN: '${{ inputs.upgrade-plan-name }}'
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+            # for now we explicitly specify this test suite.
+          - entrypoint: TestTransferTestSuite
```

Comment:
we can hard code these in as we transition, and then replace this with the dynamically generated list of tests.